package.json

```diff
@@ -51,6 +51,7 @@
     "promise-polyfill": "^3.1.0",
     "style-attr": "^1.0.2",
     "super-animejs": "^3.0.0",
+    "super-three": "^0.102.2",
```

Comment:
I'm not sure we want to pull this in -- I'm assuming this was added because of the use of `THREE.Clock`